addons/survey/views/survey_templates_management.xml

```diff
@@ -118,12 +118,29 @@
                             </div>
                             <div class="row">
                                 <div class="col-12 col-md-4 offset-md-4 text-center">
-                                    <input id="session_code" type="text" placeholder="e.g. 4812" autocomplete="off"
+                                    <input t-if="session_code" id="session_code" type="text" placeholder="e.g. 4812" autocomplete="off" class="form-control o_survey_question_text_box font-weight-bold bg-transparent text-primary text-center rounded-0 p-2 w-100" t-att-value="session_code"/>
+                                    <input t-else="" id="session_code" type="text" placeholder="e.g. 4812" autocomplete="off"
                                            class="form-control o_survey_question_text_box font-weight-bold bg-transparent text-primary text-center rounded-0 p-2 w-100"/>
                                 </div>
                                 <div class="col-12 col-md-4 offset-md-4 text-center o_survey_error text-danger pt-2" role="alert">
-                                    <span class="o_survey_session_error_invalid_code d-none">Code is incorrect.</span>
-                                    <span class="o_survey_session_error_closed d-none">Session is finished.</span>
+                                    <span t-if="error == 'survey_wrong'" class="o_survey_session_error_invalid_code">Code is incorrect.</span>
```

Comment:
If you use t-if/t-else conditions, it will mean that the blocks are not physically within the rendered template, meaning showing/hiding them later will not work. 
(To double-check).